tools/test-proxy/Azure.Sdk.Tools.TestProxy/Admin.cs

```diff
@@ -98,6 +98,15 @@ public async Task SetMatcher()
             }
         }
 
+        [HttpPost]
+        [AllowEmptyBody]
+        public async Task SetRecordingOptions([FromBody()] IDictionary<string, string> options = null)
```

Comment:
OK, so just for my understanding here for when I go to add support for this in JS. This only needs to be called once, before I start recording anything, right? And the setting will persist across all recording runs until I turn it off?

In JS we want to run the browser tests and the Node tests against the same test-proxy instance, it looks like I will need to make sure to reset the options when the browser tests are done, so that the Node tests follow redirects as expected. It's not a problem to do this, just making sure that's how it's meant to work.

Reply:
This is exactly correct. I'll add a section to the `readme.md` so you have a fallback set of details as well.